### PR TITLE
stack 0.1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by tkawachi**
stack is a cross-platform program for developing Haskell projects.
It is aimed at Haskellers both new and experienced.

### CODE REVIEW

File: Library/Formula/stack.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Stack < Formula
+  desc "The Haskell Tool Stack"
+  homepage "https://www.stackage.org/"
+  url "https://github.com/commercialhaskell/stack/releases/download/v0.1.2.0/stack-0.1.2.0-x86_64-osx.gz"
```
Comment (bfontaine) [inline Library/Formula/stack.rb]: We prefer to build from source.

Comment (tkawachi) [inline Library/Formula/stack.rb]: It requires older version of GHC than what Homebrew provides. I don't know how to use older one (is it possible?).

https://github.com/commercialhaskell/stack/wiki/Downloads#os-x
> Note: due to GHC bug 10322, building stack from source fails with GHC 7.10.1. This bug will be fixed in 7.10.2, but in the meantime, we recommend using GHC 7.8.4 on OS X.

Comment (bfontaine) [inline Library/Formula/stack.rb]: We keep some versioned formulae in [Homebrew/versions](https://github.com/Homebrew/homebrew-versions) but unfortunately GHC isn’t here, so we can’t use an older version.

Comment (bfontaine) [inline Library/Formula/stack.rb]: You should submit this to [Homebrew/binary](https://github.com/Homebrew/homebrew-binary) if we can’t build it from source. We’ll be able to move it in the core in the next version if the bug is fixed.

**Comment by DomT4** [comment]
See also: #40534

**Comment by tkawachi** [comment]
Submitted the formula to homebrew-binary. https://github.com/Homebrew/homebrew-binary/pull/242

**Pull request closed without merging**